DBImport/InsertData.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""Insert Data component.
+
+This module gets lists of Course and Department objects from ReadCourseData.py
+and insert those data into desired databse
+"""
+
+import logging
+import os
+from ReadCourseData import *
+from configparser import ConfigParser
+from pymongo import MongoClient
+from dotenv import load_dotenv
+from pathlib import Path
+
+
```

Comment:
You may put get logger here
ref: https://docs.python.org/3/library/logging.html#logging.getLogger